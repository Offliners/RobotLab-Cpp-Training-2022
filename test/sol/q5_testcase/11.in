(7/10/14)*1+9/9=-2*3=-14/(10-17)=*19/(5+6)*18=
