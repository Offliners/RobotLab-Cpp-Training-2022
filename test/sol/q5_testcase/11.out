1
-5
-3
-108
